modules/prediction/common/feature_output.cc

```diff
@@ -43,6 +45,9 @@ void FeatureOutput::Close() {
     case 3: {
       WritePredictionResult();
     }
+    case 4: {
+      WriteFrameEnv();
+    }
```

Comment:
Add explicit comment if the fall-through is expected.

Reply:
For this, I added the explicit comment in the prediction_system_gflags.cc

Reply:
Okay, I will add the comment in the next commit. Thanks.